### ISSUE TITLE
Deleting certs from certs directory does not result in certs being removed from the haproxy config.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#8366]
Rancher Versions: Build from master
Haproxy image - prachidamle/lb-service-haproxy:latest

Created LB with the following cert directories for cert and default_cert , where /certs/mycerts directory had certs for test1.com and test2.com
```
      io.rancher.lb_service.cert_dir: /certs/mycerts
      io.rancher.lb_service.default_cert_dir: /certs/test5.com
```
haproxy config has certs for test1.com , test2.com and test5 as expected

````
root@20a3e29ef9dc:/# ls /etc/haproxy/certs/current
test1.com.pem  test2.com.pem  test5.com.pem
root@20a3e29ef9dc:/#
```
From /certs/mycerts/test1.com , deleted the cert and key files ( But left the sym links privkey.pem and fullchain.pem )
```
lrwxrwxrwx 1 root root 13 Mar 31 01:24 privkey.pem -> test1.com.key
lrwxrwxrwx 1 root root 13 Mar 31 01:24 fullchain.pem -> test1.com.crt
```

Even after directory changes were synced , haproxy continues to have test1.com certs programmed in it.

Following error seen in haproxy container logs:
```
3/30/2017 6:38:33 PMtime="2017-03-31T01:38:33Z" level=debug msg="Start --- LookForCertUpdates polling dir /certs/mycerts"
3/30/2017 6:38:33 PMtime="2017-03-31T01:38:33Z" level=debug msg="lastUpdated 2017-03-31 01:35:31.03834561 +0000 UTC"
3/30/2017 6:38:33 PMtime="2017-03-31T01:38:33Z" level=debug msg="Walking dir /certs/mycerts"
3/30/2017 6:38:33 PMtime="2017-03-31T01:38:33Z" level=debug msg="Walking dir /certs/mycerts/test1.com"
3/30/2017 6:38:33 PMtime="2017-03-31T01:38:33Z" level=error msg="LookForCertUpdates: Error File  pointed by symlink /certs/mycerts/test1.com/fullchain.pem does not exist, error: lstat /certs/mycerts/test1.com/test1.com.crt: no such file or directory reading certs from cert dir  /certs/mycerts"
3/30/2017 6:38:33 PMtime="2017-03-31T01:38:33Z" level=debug msg="Walking dir /certs/test5.com"
3/30/2017 6:38:34 PMtime="2017-03-31T01:38:34Z" level=debug msg="Done --- LookForCertUpdates poll"

```

**Comment by sangeethah** [comment]
Tested with  prachidamle/lb-service-haproxy:v0.1
Deleting certs from certs directory result in certs being removed from the haproxy config as expected.

**Comment by deniseschannon** [comment]
Official image released with rancher/lb-service-haproxy:v0.7.0 with the same fix.
